bitgo-java-api/build.gradle

```diff
@@ -1,14 +1,17 @@
 apply plugin:'java'
 apply plugin:'idea'
-version='0.0.4'
+
+group = 'com.bitso'
+version='SNAPSHOT'
```

Comment:
This doesn't seem right

Reply:
Jitpack will determine version number via the github release tag, not from what we define in our build.gradle file.  This was changed here avoid any potential confusion.